Source/WebCore/html/HTMLAttachmentElement.idl

```diff
@@ -35,4 +35,6 @@
     readonly attribute DOMRectReadOnly? saveButtonClientRect;
 
     static DOMString getAttachmentIdentifier(HTMLImageElement imageElement);
+
+    readonly attribute boolean hasThumbnailOrIcon;
```

Comment:
This information should probably be exposed via `Internals.idl`, since it doesn't need to be a part of the client interface.